src/test/shell/bazel/remote/remote_build_event_uploader_test.sh

```diff
@@ -259,14 +281,35 @@ EOF
   bazel test \
       --remote_executor=grpc://localhost:${worker_port} \
       --experimental_remote_build_event_upload=minimal \
-      --build_event_json_file=bep.json \
+      --build_event_json_file=$BEP_JSON \
       //a:test >& $TEST_log || fail "Failed to build"
 
-  cat bep.json > $TEST_log
-  expect_not_log "test.sh.*bytestream://" || fail "test script is uploaded"
-  expect_log "test.log.*bytestream://" || fail "should upload test.log"
-  expect_log "test.xml.*bytestream://" || fail "should upload test.xml"
-  expect_log "command.profile.gz.*bytestream://" || fail "should upload profile data"
+  expect_bes_file_not_uploaded test.sh
+  expect_bes_file_uploaded test.log
+  expect_bes_file_uploaded test.xml
+  expect_bes_file_uploaded command.profile.gz
+}
+
+function test_upload_minimal_upload_buildlogs() {
```

Comment:
This test case failed because the release branch is missing https://github.com/bazelbuild/bazel/pull/17110. Please cherry-pick that and rebase this PR.

Reply:
The test is now redundant (check `test_upload_minimal_upload_buildlogs` below). Please remove it.